.github/workflows/release.yml

```diff
@@ -73,6 +73,40 @@ jobs:
       - name: Run tests
         run: yarn test
 
+      - name: Update Drupal install cache
```

Comment:
This is something we want to reuse I think. Maybe we could event turn it into a Github action in the future.